src/masternodes/accounts.cpp

```diff
@@ -99,25 +109,60 @@ uint32_t CAccountsView::GetBalancesHeight(CScript const & owner)
     return ok ? height : 0;
 }
 
+void CAccountsView::CreateFuturesMultiIndexIfNeeded()
```

Comment:
checking if we can add a boost test to check this function
there are some storage tests in here ->`src/test/storage_tests.cpp`
or else can run the new binary on old data and check if `CreateFuturesMultiIndexIfNeeded` works as intended.

Reply:
Thanks for pointing out, Updated and added testing for `CreateFuturesMultiIndexIfNeeded`.
Wanted to remove `Flush()` command from `CreateFuturesMultiIndexIfNeeded`, otherwise the test will fail.

Reply:
hmm. Currently `CDBWrapper::Flush()` does nothing 🤔  But I see @bvbfan has updated the code in v3.
Flush() is supposed to write to the flat file if any.
but if that's already happening at write, I guess not having `Flush` here is fine.
@prasannavl if you have more insight into this.

Reply:
Update 
further reading of the code, final resolution of the Flush should be -> https://github.com/DeFiCh/ain/blob/e63ae12e399b846c5f97dbf3cb1e771f75263255/src/flushablestorage.h#L157